documentation/modules/exploit/multi/php/jorani_path_trav.md

```diff
@@ -0,0 +1,52 @@
+## Vulnerable Application
+
+Jorani prior to 1.0.2 allows unauthenticated remote malicious users to execute arbitrary code.
+
+This is due to a lack of sanitization on language parameter, that can lead to file inclusion of ".php" files.
+
+Moreover, the log file for jorani ends with ".php" in Jorani < 1.0.2.
+
+Log poisonning is possible, so an authenticated attacker can execute arbitrary code.
+
+Finally, the controller responsible for recovering a page doesn't properly redirect requests made by Ajax.
+
+So by chaining theses 3 vulnerabilities an unauthenticated user can execute arbitrary code on the application.
+
+This module has been tested successfully on Jorani 1.0.0, Ubuntu 20.04 (x86_64) with kernel version 5.15.0-75.
+
+## Verification Steps
+1. Start `msfconsole`
+2. `use exploit/multi/php/jorani_path_trav`
+3. set `RHOSTS` and `RPORT`
+4. Confirm the target is vulnerable: `check`. The result expected is `The target appears to be vulnerable.`
+5. Default payload for the exploit will be `php/meterpreter/reverse_tcp`
+6. set `LHOST`
+7. `exploit`
+8. Confirm you have now a cmd session
+
+## Options
+
+### TARGETURI (optional)
+The path to the jorani website. By default it is empty.
+
+## Scenarios
+
+```
+msf6 exploit(multi/php/jorani_path_trav) > run
```

Comment:
Please also show which options you set or do a `show options` command output show that we can see which options you set prior to running the exploit. This helps us debug issues down the line and has solved confusions in the past so its useful to include.